addon-test-support/ilios-common/page-objects/sessions.js

```diff
@@ -7,20 +7,22 @@ import {
 } from 'ember-cli-page-object';
 
 import sessionsList from './components/sessions-grid';
+import newSession  from './components/new-session';
 
 export default create({
   visit: visitable('/courses/:courseId'),
   scope: '[data-test-course-sessions]',
   expandCollapseAllSessions: clickable('[data-test-sessions-grid-header] [data-test-expand-collapse-all]'),
   showsAllSessionsExpanded: hasClass('fa-caret-down', '[data-test-expand-all] svg'),
   expandNewSessionForm: clickable('[data-test-actions] [data-test-expand-collapse-button] button'),
-  newSession: {
-    scope: '[data-test-new-session]',
-    title: fillable('[data-test-title]'),
-    type: fillable('[data-test-type]'),
-    save: clickable('[data-test-save]'),
-    cancel: clickable('[data-test-cancel]'),
-  },
+  // newSession: {
```

Comment:
Would prefer not to leave in commented code.

Reply:
me too. removed.